1_create_database.md

```diff
@@ -0,0 +1,88 @@
+### 1. Tạo database như hình trên (sau khi tạo thì import database)
+- Create database
+```mysql
+  CREATE DATABASE `train_mysql` DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+```
+- Create table `user`
+```mysql
+CREATE TABLE `user` (
+  `id` INT(11) NOT NULL AUTO_INCREMENT,
```

Comment:
`ID` khoá chính không nên để `số âm` nên mình dùng công thức sau: 
` id INT UNSIGNED NOT NULL  AUTO_INCREMENT`

Em hiểu số 11 có ý nghĩa gì không?

Reply:
em thấy trong database mẫu để là 11 nên em làm 11
mặc đinh thì nó cũng 11 nhưng e để 11 cho nó hiểu với yêu cầu bài thôi a